ravenpy/utilities/geoserver.py

```diff
@@ -106,7 +106,7 @@ def _get_location_wfs(
         kwargs = dict(bbox=bbox)
     elif point:
         # FIXME: Remove this once OWSlib > 0.24.1 is released.
-        if Intersects is False and wfs_Point is False:
+        if Intersects is None and wfs_Point is None:
```

Comment:
I would suggest just using `if not Intersects and not wfs_Point`, which is more idiomatic.